client/src/components/PageDetails.js

```diff
@@ -39,31 +41,42 @@ class VersionNumber extends React.Component {
   }
 }
 
-export class PageDetails extends React.Component {
-  render() {
-    const { toggleSurvey, showSurvey } = this.props;
-    return (
-      <div className="pagedetails frow">
-        <div className="pagedetails__report-a-problem fcol-md-8 fcol-sm-12">
-          <Details
-            summary_content={text_maker("report_a_problem")}
-            content={<EmailFrontend template_name="report_a_problem" />}
-            persist_content={true}
-          />
-        </div>
-        <button className="btn btn-ib-primary" onClick={() => toggleSurvey()}>
-          {text_maker("feedback")}
-        </button>
-        <StatelessModal
-          title={text_maker("feedback")}
-          show={showSurvey}
-          body={<EmailFrontend top_border={false} template_name="feedback" />}
-          on_close_callback={() => toggleSurvey()}
-        />
-        <div className="pagedetails__version-number fcol-md-4 fcol-sm-6">
-          <VersionNumber />
-        </div>
-      </div>
-    );
+const PageDetails = withRouter(
+  class PageDetails extends React.Component {
+    render() {
+      const { location, toggleSurvey, showSurvey } = this.props;
```

Comment:
Total style nitpick, but I think we should prefer early returns as a convention e.g. putting

```javascript
if(location.pathname !== "/contact"){
  return null;
}
``` 

at the top.


Why? As far as I can tell, the return value is the same when `location.pathname !== "/contact"`, but it's a little tough to see that in the diffs because the indentation changed. 


Just something to keep in mind next time, merging PRs is more important than style nitpicks right now haha.